inclusive-design/wecount.inclusivedesign.ca
# Add card that links to learn page (resolves #191)

## Description
Adds card on home page of We Count website that links to Learn page and updates the svg image of all three cards on homepage.



## Steps to test

1. Got to deploy preview
2. If not already on homepage navigate to homepage
3. Ensure layout of cards match [Figma design](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=1622%3A4592) and clicking on each card navigates to their corresponding webpage.

**Expected behavior:** 
See step 3 above.
## Additional information

### Outstanding Issues with this PR

* `viewBox` needs to be adjusted to match [Figma design](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=1622%3A4592)
* The colour order of the cards has switched in the design (went form blue -> green -> yellow to blue -> yellow -> green). @uttaraghodke, @lliskovoi is it possible to switch back to the original colour order that way a bunch of files and code do not have to be rewritten.



## Related issues
closes #191
